packages/grid/demo/grid.stories.mdx

```diff
@@ -18,6 +18,8 @@ import { GRID_ROWS as ROWS } from './stories/data';
 
 # Demo
 
+> Visualize the Grid component using the `debug` toggle in the Addons panel
```

Comment:
Was this sentence meant to be a block quote?

Reply:
I was thinking of using the block-quote as a "quick tip" style area.